expand_urls.py

```diff
@@ -0,0 +1,105 @@
+import configparser
```

Comment:
Not a default package I believe. Ideally this script would work without the user needing to install anything.

(In download_better_images.py there's code to assist the user in installing requests but I was very reluctant to add it.)

Reply:
Maybe just use JSON? Afaik it doesn't require a dependency.

Reply:
`configparser` has been a core library since Python 3 days. (Before that it was called `ConfigParser`.)